junebug/stores.py

```diff
@@ -21,6 +22,11 @@ def _redis_op(self, func, id, *args, **kwargs):
         yield self.redis.expire(id, self.ttl)
         returnValue(val)
 
+    def get_key(self, channel_id, message_id):
+        '''Gets the key where a message would be stored, using the channel and
+        message ids'''
+        return '%s:%s:%s' % (channel_id, self.KEYSPACE, message_id)
```

Comment:
I did it this way to keep consistency. Currently we use `<channel-id>:properties` to store the channel properties. Should we be changing that to `channels:<channel-id>:properties`?